docs/user_guide/build_intel_boot_bin.rst

```diff
@@ -0,0 +1,811 @@
+   $sudo dd if=./u-boot-splx4.sfp of=/dev/sdz3
+    1697+1 records in
+    1697+1 records out
+    868996 bytes (869 kB, 849 KiB) copied, 0.21262 s, 4.1 MB/s
+
+.. _build_intel_boot_bin c5soc:
+
+ARRADIO/Terasic C5 SoC
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  HDL Project: :git-hdl:`here <projects/arradio/c5soc>`
+-  Linux kernel: :git-linux:`here <>`
+
+Building the Linux Kernel image and the Devicetree
+```````````````````````````````````````````````````````````````````````````````
+
+**Linux/Cygwin/WSL**
+
+.. shell::
+
+   $git clone https://github.com/analogdevicesinc/linux.git
+   $cd linux/
+   # Set architecture and compiler
+   $export ARCH=arm
+   $export CROSS_COMPILE=/path/to/arm-linux-gnueabihf
+   # Apply kconfig settings
+   $make socfpga_adi_defconfig
+   # Build the kernel
+   $make zImage
+   # Build the devicetree
+   $make socfpga_cyclone5_sockit_arradio.dtb
+
+Building the Hardware Design
+```````````````````````````````````````````````````````````````````````````````
+
+Clone the HDL repository if it does not exist yet in local directory. Then build
+the project.
+
+.. shell::
+
+   $git clone https:github.com/analogdevicesinc/hdl.git
+   $cd hdl/projects/arradio/c5soc
+   $make
+
+After the design was built, the resulted SRAM Object File (.sof) file shall be
+converted to a Raw Binary File (.rbf).
+
+If you skipped the last section, ensure to set the architecture and cross
+compiler environment variables.
+
+.. caution::
+
+   Pay attention to directory changes to where the commands are run from,
+   and always confirm with ``pwd`` to show the current path at you terminal.
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/arradio/c5soc ; pwd
+    ~/hdl/projects/arradio/c5soc
+   $quartus_cpf -c --hps -o bitstream_compression=on \
+   $    ./arradio_c5soc.sof soc_system.rbf
+
+Building the Preloader and Bootloader Image
+```````````````````````````````````````````````````````````````````````````````
+
+This flow applies starting with release 2021_R1 / Quartus Pro version 20.1.
+For older versions of the flow see previous versions of this page on wiki
+:dokuwiki:`Altera SOC Quick Start Guide <resources/tools-software/linux-software/altera_soc_images>`.
+
+In HDL project directory, create the software/bootloader folder and clone the
+``u-boot-socfpga`` image. Before that create a new BSP settings file.
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/arradio/c5soc ; pwd
+    ~/hdl/projects/arradio/c5soc
+   $mkdir -p software/bootloader
+   $embedded_command_shell.sh bsp-create-settings --type spl \
+   $    --bsp-dir software/bootloader \
+   $    --preloader-settings-dir "hps_isw_handoff/system_bd_sys_hps" \
+   $    --settings software/bootloader/settings.bsp
+   $cd software/bootloader ; pwd
+    ~/hdl/projects/arradio/c5soc/software/bootloader
+   $git clone https://github.com/altera-opensource/u-boot-socfpga.git
```

Comment:
Pay attention to the u-boot branch used - on de10nano example you check out a specific branch, but here not. 
For example 'socfpga_v2021.10' is used for Quartus 22.1 Std (so u-boot tag has nothing to do with Quartus version).

Reply:
This was taken from Wiki. I will fix it.

Reply:
It should be corrected also on wiki, if the page will remain publicly accessible.